src/pudl/metadata/resources/mcoe.py

```diff
@@ -183,12 +183,10 @@
         for freq in AGG_FREQS
     }
     | {
-        f"mcoe_generators_{freq}": {
+        f"out_eia__{freq}_generators": {
```

Comment:
We decided to add all generator attributes to this table so it's a one stop shop for users.